### ISSUE TITLE
[Electron debugging] Is there any remote debugging configuration?

### ISSUE THREAD

**Issue by anaselhajjaji** [microsoft/vscode-recipes#228]
The use case is:
Having the application running in a raspberry pie and the source code in a Windows Machine.

Question:
I want to debug in VS Code the Electron Renderer of the application that is running in the raspberry pie. Is it possible?

**Comment by connor4312** [comment]
Yes, you can start your process with [`--inspect`](https://nodejs.org/api/cli.html#cli_inspect_host_port) and [attach to it](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_attaching-to-nodejs). Or, more easily, use [remote ssh](https://code.visualstudio.com/docs/remote/ssh).
